src/api/precautionApi.tsx

```diff
@@ -84,6 +84,13 @@ const getUserActivePrecautionEndDate = async (userId: string, type: string) => {
     type,
   });
 };
+
+const getUsersForPrecaution = async () =>{
+  return await api.get(`Precaution/getUsersForPrecaution`).catch((error)=>{
+    throw new Error(error);
```

Comment:
1. Stack trace of error is lost, because instead of re-throwing catched `error`, you create a new error instead.
2. Why actually do you use catch to throw error again? Remove whole `.catch` block